common/scala/src/main/scala/whisk/core/entity/WhiskAction.scala

```diff
@@ -313,6 +313,9 @@ object WhiskAction extends DocumentFactory[WhiskAction] with WhiskEntityQueries[
 
   override val cacheEnabled = true
 
+  val requireWhiskAuthAnnotation = "require-whisk-auth"
```

Comment:
I don't think these values belong here since `WhiskAction` does not know what a Web Action is.

Reply:
since a web action is actually a "kind" of `WhiskAction, and since the annotation is set within the context of an action create/update, imho it makes sense to define all action annotation constants as part of `WhiskAction`

Reply:
I also suggested below that we should instead consolidate all the annotations into a singleton - they're scattered right now.